main.js

```diff
@@ -36,8 +36,8 @@ function makeRequest(fn, uri, options, callback) {
 }
 
 Trello.prototype.makeRequest = function (requestMethod, uri, options, callback) {
-    if (options === 'undefined') {
-        options = {}
+    if (typeof options === 'undefined') {
```

Comment:
Wouldn't `options = options || {}` do the same? Because the signature of the function has the callback as the last argument, one might want to call it using: `makeRequest("get", "/1/cards", null, (err, result) => result)`